pkg/lib/cockpit.js

```diff
@@ -26,9 +26,6 @@ import { event_mixin } from './event-mixin';
 import { url_root, transport_origin, calculate_application, calculate_url } from './location';
 import { ParentWebSocket } from './parentwebsocket';
 
-/* injected by tests */
-var mock = mock || { }; // eslint-disable-line no-use-before-define, no-var
```

Comment:
Lets split this up :)

Reply:
This is in a two-line commit on its own.  Or do you mean move it to a separate PR that we land before landing this PR?